static/js/click_handlers.js

```diff
@@ -399,6 +399,11 @@ $(function () {
 
     });
 
+    // on the end of the modified-message fade in, remove the fade-in-message class.
+    $("body").one("webkitAnimationEnd oanimationend msAnimationEnd animationend", ".fade-in-message", function () {
```

Comment:
Well if you want you could do `ui.js`.  Basically I don't think this belongs with the click-handler collection :)